src/ProjectModule/Components/ProjectList/ProjectList.module.css

```diff
@@ -0,0 +1,45 @@
+.title{
+    background-color: #F5F5F5;
+    }
+    .borderless input{
+        border: none; /* Remove the border */
+        outline: none; /* Remove the outline when focused */
```

Comment:
outlines are useful when using the keyboard for navigation, we shouldn't remove them unless we will provide another style for them to indicate where we are in the page.